server/account/account.go

```diff
@@ -1,6 +1,10 @@
 package account
 
 import (
+	"database/sql/driver"
```

Comment:
At first it seemed a bit awkward to have this import, but it's a standard library import so I think it's OK.